src/hydro_serving_grpc/writer/writer.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package hydrosphere.writer;
+option java_package = "io.hydrosphere.writer";
+
+import "google/protobuf/empty.proto";
+
+
+message WriteOneRequest {
+    int64 model_version = 1;
+    string feature = 2;
+    bool check = 3;
+    int64 timestamp = 4;
+    int64 uid = 5;
+    double value = 6;
+    string description = 7;
+}
+
+message WriteManyRequest {
```

Comment:
In my opinion the way you introduced multiple requests in single message is not quite reliable.
How do you check, for instance, that `len(model_version) == len(feature) == ... == len(description)` ?

Wouldn't it be nicer and simpler to define WriteManyRequest as `repeated WriteSingleRequest`?

Reply:
Yeah, you're right, I didn't know about `repeated` functionality. I'll check it out.

Reply:
Thanks for suggestion, I've refactored the code.